### DISCUSSION TITLE
mogrify png to tif with transparency as extra alpha channel, no transparency

### DISCUSSION THREAD

**Discussion by dt192** [ImageMagick/ImageMagick#7134] [Help] [answered]
### ImageMagick version

ImageMagick-7.1.1-29-portable-Q16-HDRI-x64

### Operating system, version and so on

Win 10

### Description

When I convert png to tif using xnview, they show as transparent thumbnails in file explorer, but when i open them in photoshop they show as RGB plus an alpha channel. This is what I want, however with ImageMagick when I open the tif, the transparency is embedded with no 4th channel.

I know I can pull the data out using Photoshop, but I'd rather just have them set the way I want them as default. Can this be done?  

`D:\portable\ImageMagick\mogrify.exe -format tif *.png` currently using this and think I have tried all the variations of `-alpha X`. I thought `deactivate` would be correct, but that made no difference.

![2024-02-26_12-20](https://github.com/ImageMagick/ImageMagick/assets/2612048/50f51a0c-d144-4ffa-a4b6-bd7b5567cd79)

**Comment by dlemstra** [comment]
Please share your input image.

**Reply by amiramir2124** [reply]
https://khanephotoshop.com/

photoshop tutorials
and
Photoshop products

**Comment by dt192** [comment]
[images.zip](https://github.com/ImageMagick/ImageMagick/files/14404934/images.zip)

**Comment by fmw42** [comment, marked as answer]
This works for me using the -define for how the alpha channel will be stored in the tiff file. It should also work with mogrify

`magick test.png -define tiff:alpha=unspecified x.tiff`

[x.tiff.zip](https://github.com/ImageMagick/ImageMagick/files/14408761/x.tiff.zip)

**Reply by dt192** [reply]
It does work with mogrify. Thanks for the help :)

**Comment by amiramir2124** [comment]
https://khanephotoshop.com/

photoshop tutorials
and
Photoshop products
